src/content/app/species/components/species-title-area/SpeciesTitleArea.module.css

```diff
@@ -47,6 +47,7 @@
 }
```

Comment:
currently:

![image](https://github.com/Ensembl/ensembl-client/assets/6834224/7e33dcb7-ef46-4ad1-bee4-940341d0eaec)

to fix, add:
`align-items: center;` to `.assemblyCountWrapper`

Reply:
I don't see it like this on Chrome. Which browser does it appear like this in? Also, when I add centre it the padding reverts to what it was before.
![Screenshot 2024-03-18 at 17 18 49](https://github.com/Ensembl/ensembl-client/assets/32512909/7c8e8279-b55c-48c3-9b07-8e8a6bd35fbf)

Reply:
I checked this in the wrong branch so please ignore the above comment